### ISSUE TITLE
JAWs 2022 does not always respect aria-live regions in Electron apps

### ISSUE THREAD

**Issue by sivakusayan** [FreedomScientific/VFO-standards-support#602]
### Summary

This seems to be a regression from JAWs 2021.  Although Electron is based in Chromium, I am not able to reproduce in Chrome.

1. Clone [electron-quick-start](https://github.com/electron/electron-quick-start) to a local folder.
2.  Use the instructions in the repository README to start the application.
3.  Ctrl+Shift+I to open developer tools.
4.  In the console, set `window.location = "https://www.freedomscientific.com/SurfsUp/AriaLiveRegionsAtomic.htm"`
5.  Open JAWs 2022.

### Expected result

JAWs 2022 reads the updated text.

### Actual result

JAWs 2022 does not read the updated text.

#### Technical Details
When debugging using Accessibility Insights for Windows, I am correctly seeing the [UIA LiveRegionEvent](https://www.w3.org/TR/core-aam-1.2/#mapping_events_visibility) firing, which is why I am writing this bug up here first rather than in the Electron repository. I am also seeing NVDA correctly respect the live region.

I mentioned "does not always" in the title because JAWs 2022 seems to work fine with Microsoft Teams for some reason. Chat messages, which use live regions under-the-hood, are correctly respected and read out by JAWs 2022, so I'm not sure why things are working differently here.

#### JAWS version and build number
JAWs Version 2022.2112.24 ILM
 
#### Operating System and version
Microsoft Windows 10 Enterprise
Version: 10.0.19042 Build 19042

#### Browser and version:
Electron Version 16.0.8
(This version of Electron uses Chromium 96.0.4664.110)

**Comment by sivakusayan** [comment]
Adding more details here.

I used Chromium's [ax_dump_events](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/tools/accessibility/inspect/) tool to log what events were being fired in this workflow. As far as I can tell, I think Electron is [firing the events that it should be](https://www.w3.org/TR/core-aam-1.2/#mapping_events_visibility), and satisfies UIAutomation and IAccessible2 APIs.

[AXEventsDump.txt](https://github.com/FreedomScientific/VFO-standards-support/files/8345929/AXEventsDump.txt)
